packages/next/server/web/adapter.ts

```diff
@@ -8,6 +8,7 @@ import { NextResponse } from './spec-extension/response'
 import { relativizeURL } from '../../shared/lib/router/utils/relativize-url'
 import { waitUntilSymbol } from './spec-extension/fetch-event'
 import { NextURL } from './next-url'
+import { stripInternalSearchParams } from '../utils'
```

Comment:
I think this import might cause all of constants.ts to be included in the edge runtime. Could you move `stripInternalSearchParams` and `stripInternalQueries` into a separate file just to be sure 👍